telemetry/aiwrapper_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright Microsoft. All rights reserved.
```

Comment:
don't need license headers, the license.txt in the root covers everything

Reply:
You meant `license/cla — All CLA requirements met.` in pipeline covers all? So, we do not need to add this line anymore for new files, but leave it in existing files?

Reply:
unless the file has a _different_ license, all files in this repo are covered by https://github.com/Azure/azure-container-networking/blob/master/LICENSE